bigtable/tests/system.py

```diff
@@ -269,12 +268,22 @@ def test_create_instance_w_two_clusters(self):
                          alt_cluster_2.name,
                          Config.CLUSTER.name}.issubset(found))
 
+        # test get_cluster_states for a table in instance
+        expected_result = {ALT_CLUSTER_ID_1: "READY",
+                           ALT_CLUSTER_ID_2: "READY"
+                           }
+        temp_table_id = 'test-get-cluster-states'
+        temp_table = instance.table(temp_table_id)
+        temp_table.create()
+        result = temp_table.get_cluster_states()
+        self.assertEqual(result, expected_result)
```

Comment:
Please verify that `replication_state` value returned by the server is one of the possible `replication_state` valuses (`int` value 0 - 4)

Reply:
For better comprehension please refactor this test in a separate helper method and call the method here